### ISSUE TITLE
[Anon] Bug report: Imperium - Dark Angels.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#2099]
**File:** Imperium - Dark Angels.catz

**BattleScribe version:** 2.01.08

**Platform:** Windows

**Dropbox:** No

**Description:** When selecting weapons for a tactical Marines Sgt. you can`t chose to substitute Bolt pistol and Bolter. you can unselect those but the file proclaims an invalid choice if you select more than one substitute. 
i.e. you can select a plama pistol instead of the Bolt pistol but get an error. => unselecting the Bolt gun removes that error. same is the other way around.
IF you select a plasma pistol and, after unselecting the bolt gun, select a Chainsword you get an error again. etc.
basically it is currently either Boltpistol and Boltgun or only one other weapon choice.
